channeldb/invoices.go

```diff
@@ -546,6 +546,61 @@ func (d *DB) LookupInvoice(paymentHash [32]byte) (Invoice, error) {
 	return invoice, nil
 }
 
+// FetchAllInvoicesWithPaymentHash returns all invoices and their payment hashes
+// currently stored within the database. If the pendingOnly param is true, then
+// only unsettled invoices and their payment hashes will be returned, skipping
+// all invoices that are fully settled.
+func (d *DB) FetchAllInvoicesWithPaymentHash(pendingOnly bool) (
+	[]Invoice, []lntypes.Hash, error) {
```

Comment:
It seems that this code is changed later on in this pr, maybe because of a rebase mistake?